build.assets/windows/build.ps1

```diff
@@ -484,23 +501,12 @@ function Write-Version-Objects {
     & $GoWinres simply --no-suffix --arch amd64 `
         --file-description "Teleport tctl administrative tool" `
         --original-filename tctl.exe `
-        --copyright "Copyright (C) $Year Gravitational Inc." `
+        --copyright "Copyright (C) $Year Gravitational, Inc." `
         --icon "$TeleportSourceDirectory\e\windowsauth\installer\teleport.ico" `
         --product-name Teleport `
         --product-version $TeleportVersion `
         --file-version $TeleportVersion `
         --out "$TeleportSourceDirectory\tool\tctl\resource.syso"
-
-    # generate windowsauth version info (note the --admin flag, as the installer must run as admin)
-    & $GoWinres simply --no-suffix --arch amd64 --admin `
```

Comment:
Why was this removed?! This is necessary in order for the installer to properly request admin permissions.